### DISCUSSION TITLE
Openai assistant integration

### DISCUSSION THREAD

**Discussion by ai003** [vercel/community#6996] [Help]
### Summary

I am trying to integrate the openai assistant with file reading capabilities and create a thread in this repository: https://github.com/vercel/ai-chatbot
But I can't find the backend logic to modify the API call to modify this repository. Any assistance is appreciated.

### Example

https://github.com/vercel/ai-chatbot

### Steps to Reproduce

Add openai key

**Comment by versecafe** [comment]
@ai003 It's handled through an action that can be found here:
`https://github.com/vercel/ai-chatbot/blob/main/lib/chat/actions.tsx`
